src/crypto/mode/cfb.js

```diff
@@ -153,21 +254,11 @@ function aesDecrypt(algo, key, ct, iv) {
 }
 
 function xorMut(a, b) {
-  for (let i = 0; i < a.length; i++) {
+  for (let i = 0; i < Math.min(a.length, b.length); i++) {
```

Comment:
This runs `Math.min(a.length, b.length)` on every iteration of the loop. The browser might optimize it, but it's better to be safe and hoist it out into a `const`